src/exercises/day-3/safe-ffi-wrapper.md

```diff
@@ -2,7 +2,7 @@
 
 Rust has great support for calling functions through a _foreign function
 interface_ (FFI). We will use this to build a safe wrapper for the `libc`
-functions you would use from C to read the filenames of a directory.
+functions you would use from C to read the names of files in a directory.
```

Comment:
This change is unrelated and has already been reviewed in #1286. You should create a new branch for every change and avoid making changes "on top" of each other:

```suggestion
functions you would use from C to read the filenames of a directory.
```